modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -13,11 +13,10 @@ The {oadp-first} 1.3.0 release notes lists new features, resolved issues and bug
 
 .Velero built-in DataMover
 
-OADP 1.3 includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses Kopia as the uploader mechanism to read the snapshot data and to write to the Unified Repository.
-
+:FeatureName: Velero built-in DataMover
+include::snippets/technology-preview.adoc[]
 
-//:FeatureName: Velero built-in DataMover
-Velero built-in DataMover is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+OADP 1.3 includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses Kopia as the uploader mechanism to read the snapshot data and to write to the Unified Repository.
```

Comment:
Two things:

The body of this content talks about ". . . a build-in Data Mover. . ." and "The built-in Data Mover. . ." but the feature itself is a "Velero built-in Data Mover." We don't mention in the body that it's Velero at all. Maybe that isn't important, or maybe it should be included. 

The second thing: your commits need squashed.

If you don't want to include "Velero" in the intro text that's fine, but at the very least the commits need squashed. Thanks!